### PR TITLE
[YLP-956] migrate to logrus

### PR DESCRIPTION
**Pull request by Nas84**
In order to facilitates the effort of standardization of our logs, we require to use Logrus (an api proven library) because we create our mongo client inside this libraries. Logrus  helps a lot  in configuration of the logger. 
- use the new log lib
- update changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 Go-Common is the library of common functions for Tidepool's Go-based applications
 
+## 1.1.0 - 2021-08-19
+### Changed
+- YLP-956: migrate log to lorgus libraries
```
Comment (ydrisR) [inline CHANGELOG.md:7]: Why?

File: clients/mongo/mongo_test.go [hunk 2]
```diff
@@ -239,7 +239,9 @@ func TestConnection(t *testing.T) {
 		// if mongo connexion information is provided via env var
 		config.FromEnv()
 	}
-	logger := log.New(os.Stdout, "mongo-test ", log.LstdFlags|log.LUTC|log.Lshortfile)
+	//logger := log.New(os.Stdout, "mongo-test ", log.LstdFlags|log.LUTC|log.Lshortfile)
```
Comment (ydrisR) [inline clients/mongo/mongo_test.go]: If we migrate it's better to remove the line rather than commenting it out.

**Review by ydrisR — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (Nas84) [reply CHANGELOG.md:7]: i added the explanation in the JIRA ticket and inside this PR

**Review by ydrisR — APPROVED** [review]
LGTM

**Pull request merged**